### PR TITLE
Remove organization dependency and improve display ad UX

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This PR removes the organization dependency for display ads which was confusing admins due to the fact that organization attribution was never fully implemented.

In the future we may implement further features that make use of this, or we could remove it altogether, but this should remove confusion.

In implementation I wanted to fix some other UX issues with display ads and ad clarity. So now we have more instructions, and we also have a preview of the display ad right there in the admin section.

I also changed the "search" UX to include multiple columns instead of just searching by organization name.

**Creating new ad:**

Screen Shot 2021-10-04 at 10 08 02 AM

**Displaying saved ad:**

Screen Shot 2021-10-04 at 10 12 36 AM

Along the way I noticed that we still have the "help link", but it generally links to the wrong places. I'm not sure what the intentions of this are, but I commented it out for now.

## Related Tickets & Documents

https://github.com/forem/rfcs/issues/287

## Added/updated tests?

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

### CODE REVIEW

**Comment by juliannatetreault** [comment]
> Along the way I noticed that we still have the "help link", but it generally links to the wrong places. I'm not sure what the intentions of this are, but I commented it out for now.

👋 

Per [this comment](https://github.com/forem/rfcs/issues/238#issuecomment-927721603) from @Ridhwana, it seems as though the plan is to audit and then update or remove the "help links," depending on whether or not we have up-to-date, useful documentation for each individual link. The full issue goes into a bit more detail, which may be helpful, @benhalpern!

**Comment by benhalpern** [comment]
Thanks @juliannatetreault. This tidbit is definitely out of scope of this PR, so if the right thing is to not comment that out and just do nothing, that makes sense.

I commented it out (not removed) just because it's literally pointing to a 404 right now. But if that's not the right thing to do I can remove that change and stick to the meet of this PR. Let me know @Ridhwana — IMO the _temporary_ removal of this pending at least fixing the links kind of makes sense, but I'm cool with whatever you want to do.

Screen Shot 2021-10-04 at 9 17 23 PM

**Comment by Ridhwana** [comment]
> Let me know @Ridhwana — IMO the _temporary_ removal of this pending at least fixing the links kind of makes sense, but I'm cool with whatever you want to do.


@benhalpern this is good for now thanks 👍🏽

**Review by Ridhwana — APPROVED** [review]

File: app/controllers/admin/display_ads_controller.rb [hunk 1]
```diff
@@ -6,13 +6,13 @@ class DisplayAdsController < Admin::ApplicationController
...
     def index
       @display_ads = DisplayAd.order(id: :desc)
-        .joins(:organization)
-        .includes([:organization])
         .page(params[:page]).per(50)
 
       return if params[:search].blank?
 
-      @display_ads = @display_ads.where("organizations.name ILIKE :search", search: "%#{params[:search]}%")
+      @display_ads = @display_ads.
```
Comment (citizen428) [inline app/controllers/admin/display_ads_controller.rb:13]: For anyone else who wonders, this is a small table (currently 29 items on DEV), so even inefficient queries are probably ok.

**Review by citizen428 — APPROVED** [review]

**Pull request merged**